### PR TITLE
[TRIVIAL] Document secure_gateway for gRPC

### PR DESCRIPTION
**Pull request by cjcobb23**
It is recommended to add a `secure_gateway` section to `[port_grpc]` when the server is an ETL source for a reporting mode server. This commit adds a few lines to the example config to explain such. There are no code changes.

---
This change is [Reviewable](https://reviewable.io/reviews/ripple/rippled/3762)

### CODE REVIEW

File: cfg/rippled-example.cfg [hunk 1]
```diff
@@ -939,6 +939,19 @@
 #   connection info is specified under the [ledger_tx_tables] config section;
 #   see the Database section for further documentation.
 #
+#   Each ETL source specified must have gRPC enabled (by adding a [grpc_port]
```
Comment (intelliot) [inline cfg/rippled-example.cfg:942]:
```suggestion
#   Each ETL source specified must have gRPC enabled (by adding a [port_grpc]
```

File: cfg/rippled-example.cfg [hunk 2]
```diff
@@ -939,6 +939,19 @@
...
+#   section to the config). It is recommended to add a secure_gateway entry to
+#   the gRPC section, in order to bypass the server's rate limiting mechanism.
```
Comment (intelliot) [inline cfg/rippled-example.cfg:944]:
```suggestion
#   the gRPC section, in order to bypass the server's rate limiting.
```

File: cfg/rippled-example.cfg [hunk 3]
```diff
@@ -939,6 +939,19 @@
...
+#   Note, this section needs to be added to the config of the ETL source, not
```
Comment (intelliot) [inline cfg/rippled-example.cfg:945]:
```suggestion
#   This section needs to be added to the config of the ETL source, not
```

File: cfg/rippled-example.cfg [hunk 4]
```diff
@@ -939,6 +939,19 @@
...
+#   the config of the reporting node. An example is listed below, where the
```
Comment (intelliot) [inline cfg/rippled-example.cfg:946]:
```suggestion
#   the config of the reporting node. In the example below, the
```

File: cfg/rippled-example.cfg [hunk 5]
```diff
@@ -939,6 +939,19 @@
...
+#   reporting server is running at 127.0.0.1. Note, multiple IPs can be
```
Comment (intelliot) [inline cfg/rippled-example.cfg:947]:
```suggestion
#   reporting server is running at 127.0.0.1. Multiple IPs can be
```

**Review by intelliot — COMMENTED** [review]
I think the section is port_grpc, not grpc_port.

I suggested removing a few extra words, but it's not a big deal either way :)

**Comment by cjcobb23** [comment]
> I think the section is port_grpc, not grpc_port.
> 
> I suggested removing a few extra words, but it's not a big deal either way :)

Good catch, thank you @intelliot . I applied the rest of your suggestions too.

**Review by nbougalis — APPROVED** [review]

**Comment by rasloun** [comment]
How can i unsubscribe???????

**Review by carlhua — APPROVED** [review]

**Review by mDuo13 — APPROVED** [review]

**Pull request closed without merging**